composer.json

```diff
@@ -21,7 +21,8 @@
         "cakephp/cakephp": "^4.4",
         "cakephp/cakephp-codesniffer": "^4.0",
         "firebase/php-jwt": "^6.2",
-        "phpunit/phpunit": "^8.5 || ^9.3"
+        "phpunit/phpunit": "^8.5 || ^9.3",
+        "symfony/cache": "^5.4 || ^6.0"
```

Comment:
Why do we need this dependency?

Reply:
see the discussion here:

https://github.com/cakephp/authentication/pull/535#discussion_r872972048

this was rebased / reopened as github closed it beeing stale for a while.

Reply:
OK, so its the PSR6 vs PSR16 compatibility shims that we actually need. While I don't love the additional dependency I also don't relish maintaining adapter code for a low-frequency use feature. We'll should include the additional dependencies required in the documentation and raise an error with instructions to install the required dependencies if they don't exist.